### PR TITLE
Check nodes and cluster health with kube clientset

### PR DESCRIPTION
**Pull request by crazytaxii**
### What type of PR is this?

/kind feature

### What this PR does / why we need it:

Check nodes and cluster health with kube clientset after deploying.

### Does this PR introduced a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by lixd** [comment]
@crazytaxii 检测没过

**Comment by crazytaxii** [comment]
> @crazytaxii 检测没过

有没有权限重跑？

**Comment by lixd** [comment]
> > @crazytaxii 检测没过
> 
> 有没有权限重跑？

还是不行，有个 test 没过，> > @crazytaxii 检测没过
> 
> 有没有权限重跑？

https://github.com/kubeclipper/kubeclipper/actions/runs/10384969333/job/28806111144?pr=750#step:4:204

**Comment by crazytaxii** [comment]
> > > @crazytaxii 检测没过
> > 
> > 
> > 有没有权限重跑？
> 
> 还是不行，有个 test 没过，> > @crazytaxii 检测没过
> 
> > 有没有权限重跑？
> 
> https://github.com/kubeclipper/kubeclipper/actions/runs/10384969333/job/28806111144?pr=750#step:4:204

应该测试代码本身有问题；CI 写的不好，用 make 错误被隐藏了

File: pkg/scheme/core/v1/k8s/kubeadm.go [hunk 1]
```diff
@@ -865,31 +877,44 @@ func (stepper *Health) Uninstall(ctx context.Context, opts component.Options) ([
...
-	return fmt.Errorf("some nodes are not ready")
+
+	logger.Info("all nodes in the cluster are ready")
+	return
 }
 
-// k8s own component(kube-systemd) pod status is running
-func (stepper *Health) checkPodStatus(ctx context.Context, opts component.Options) error {
-	ec, err := cmdutil.RunCmdWithContext(ctx, opts.DryRun, "bash", "-c", `kubectl get po -n kube-system | grep -v Running`)
+func (stepper *Health) kubeSystemPodsReady(ctx context.Context, opts component.Options) (err error) {
```
Comment (lixd) [inline pkg/scheme/core/v1/k8s/kubeadm.go:900]: allNodesReady 和 kubeSystemPodsReady 怎么看起来是一样的。

**Review by lixd — APPROVED** [review]
/lgtm
/appvore

**Pull request merged**